app/lib/zendesk.server.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { getEnvOrDie } from './env.server'
+import { getBasicAuthHeaders } from './headers.server'
+
+interface ZendeskRequest {
+  requester: Requester
+  subject: string
+  comment: RequestComment
+}
+
+interface Requester {
+  email: string
+  name: string
+}
+
+interface RequestComment {
+  body: string
+}
+
+export async function postToZendesk(request: ZendeskRequest) {
```

Comment:
Suggest more descriptive function name:

```suggestion
export async function postZendeskRequest(request: ZendeskRequest) {
```